src/views/components/textarea/index.njk

```diff
@@ -0,0 +1,40 @@
+---
+title: Textarea
+tags: components
+description: A textarea lets users enter multiple lines of text, such as when answering an open-ended question.
+layout: default
+eleventyNavigation:
+  key: Textarea
+  section: components
+---
+
+<header class="mb-8">
+    <div class="text-grey-2 text-lg mb-1">Components</div>
+    <h1 class="mb-6">{{title}}</h1>
+    <p>{{description}}</p>
+</header>
+
+{% from "example.njk" import example %}
+{{ example({group: "components", item: "textarea", example: "default", html: true, open: false, size: "m"}) }}
+
+<section class="my-8">
```

Comment:
In Vue and Svelte etc. there's the concept of slots which would be quite handy here, I think, as these sections will most likely be reused across our docs. Is there any way we could mimic this with nunjucks?

Reply:
Coming from a React background, this is exactly my problem with njk—you can't compose components together. Unless you can and I haven't figured it out yet but I suspect the syntax would be some horrible workaround using `set` and the params of `macro`.